kaggle_requirements.txt

```diff
@@ -0,0 +1,139 @@
+altair>=5.4.0
```

Comment:
super optional: seem sorta alphabetically ordered, is this intentional?
do we want to add a comment at the top to enforce it? ngl it is easier to read.